### PR TITLE
clean up auth code sample a little

### PR DESCRIPTION
**Pull request by DarylThayil**
Cleaned up the auth code sample a little to be a little more idiomatic to what an express app looks like.

### CODE REVIEW

**Review by sangonzal — APPROVED** [review]
Looks good.

**Review by sameerag — APPROVED** [review]
lgtm. Eventually we may also want to write code with "await" and "async".

**Comment by technical-boy** [comment]
LGTM!

File: samples/msal-node-auth-code/index.js [hunk 1]
```diff
@@ -54,9 +50,11 @@ function acquireToken(req, res){
 
     pca.acquireTokenByCode(tokenRequest).then((response) => {
         console.log(JSON.stringify(response));
-        res.send(200);
+        res.status(200).send(JSON.stringify(response));
```
Comment (jasonnutter) [inline samples/msal-node-auth-code/index.js:53]: I believe this can be `res.json(response)`.

File: samples/msal-node-auth-code/index.js [hunk 2]
```diff
@@ -2,16 +2,12 @@
...
-
-//initialize express
 const express = require("express");
-const app = express();
-const port = 3000;
-app.get('/', (req, res) => redirectToAzureAd(req, res));
-app.get('/redirect', (req, res) =>  acquireToken(req, res));
+const msal = require('@azure/msal-node');
+
+const SERVER_PORT = 3000;
```
Comment (jasonnutter) [inline samples/msal-node-auth-code/index.js:8]: General best practice is to look for `process.env.PORT`, and fallback to a hard-coded one:

```suggestion
const SERVER_PORT = process.env.PORT || 3000;
```

**Comment by DarylThayil** [comment]
@jasonnutter  feedback resolved. 
Added same build scripts to device code as well, no sample updates made there

**Pull request merged**